content/Lab1/60_Testing_Traffic/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Testing Traffic"
-weight: 80
+weight: 60
 ---
 In this section, we will test traffic to the test machines that are behind an AWS network load balancer via the firewalls.
 
-[Testing traffic - Setup Validation](../80_Testing_Traffic/1_Testing_Traffic.md)
\ No newline at end of file
+[Testing traffic - Setup Validation](../60_Testing_Traffic/1_Testing_Traffic.md)
```

Comment:
please rename the respective md titles so that it matches with this text e.g. 1_network_resources.md title should be AWS Network Resources - Description